student.rb

```diff
@@ -1,11 +1,18 @@
 require_relative 'person'
 
 class Student < Person
```

Comment:
- [ ] Kindly add the method for associating the student to a classroom. This method should serve as the `belongs-to` end of the relationship. See example.

  ```
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
  ```